### DISCUSSION TITLE
Wakacyjne daily

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#382] [Organisation of work]
## Kiedy się spotykamy na daily?

**Comment by emkarcinos** [comment]
Proponuję czwartek wieczorem, na przykład o 20:00

**Reply by mattyl006** [reply]
Mi to pasi

**Reply by itchystone** [reply]
A co powiecie na soboty o 12:00?

**Reply by emkarcinos** [reply]
Mi weekendy ogólnie średnio pasują, ale jeżeli nie da rady to może być

**Reply by mattyl006** [reply]
Myślę że soboty 12;00 to dosyć śliski termin, lepiej już jakiś wieczorek, czy jakaś tam pora, gdzie zazwyczaj się wraca do domu

**Reply by itchystone** [reply]
To mogą być czwartki, a moglibyście późnym wieczorem, np. 21:30?

**Reply by mattyl006** [reply]
Tak, myślę że takie czwartki byłyby bardziej prawdopodobne, że będziemy przy kompie, 21;30 git

**Reply by emkarcinos** [reply]
Czwartek 21:30 pasuje.
